<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindowTask</class>
 <widget class="QMainWindow" name="MainWindowTask">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>545</width>
    <height>405</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Практикум</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <layout class="QGridLayout" name="gridLayout"/>
    </item>
    <item row="0" column="1">
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QTreeView" name="treeView">
       <property name="minimumSize">
        <size>
         <width>200</width>
         <height>100</height>
        </size>
       </property>
       <property name="showDropIndicator" stdset="0">
        <bool>false</bool>
       </property>
       <property name="iconSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
       <property name="indentation">
        <number>20</number>
       </property>
       <property name="uniformRowHeights">
        <bool>true</bool>
       </property>
       <property name="animated">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QWebEngineView" name="webView" native="true">
       <property name="url" stdset="0">
        <url>
         <string>about:blank</string>
        </url>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>545</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Практикум</string>
    </property>
    <addaction name="loadCurs"/>
    <addaction name="actionSave"/>
    <addaction name="separator"/>
    <addaction name="actionClose"/>
    <addaction name="separator"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>Задание</string>
    </property>
    <addaction name="do_task"/>
    <addaction name="checkTask"/>
    <addaction name="action_4"/>
    <addaction name="actionReset"/>
    <addaction name="actionTested"/>
   </widget>
   <widget class="QMenu" name="menuKurs">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="title">
     <string>Редактирование</string>
    </property>
    <addaction name="actionAdd"/>
    <addaction name="actionRemove"/>
    <addaction name="actionEdit"/>
    <addaction name="addDeep"/>
    <addaction name="separator"/>
    <addaction name="actionSaveK"/>
    <addaction name="actionSaveKas"/>
    <addaction name="actionNewK"/>
    <addaction name="separator"/>
    <addaction name="actionup"/>
    <addaction name="actionDown"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
   <addaction name="menuKurs"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="loadCurs"/>
   <addaction name="do_task"/>
   <addaction name="checkTask"/>
   <addaction name="separator"/>
   <addaction name="actionup"/>
   <addaction name="actionDown"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="loadCurs">
   <property name="text">
    <string>Загрузить курс</string>
   </property>
  </action>
  <action name="checkTask">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Проверить</string>
   </property>
  </action>
  <action name="action_3">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Сбросить</string>
   </property>
  </action>
  <action name="action_4">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Подсказка</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="text">
    <string>Закрыть</string>
   </property>
  </action>
  <action name="do_task">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Выполнять</string>
   </property>
   <property name="visible">
    <bool>false</bool>
   </property>
  </action>
  <action name="actionSave">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Сохранить как...</string>
   </property>
  </action>
  <action name="actionReset">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Вернуться к исходной</string>
   </property>
  </action>
  <action name="actionTested">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Вернуться к проверенной</string>
   </property>
  </action>
  <action name="actionAdd">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Копировать</string>
   </property>
  </action>
  <action name="actionRemove">
   <property name="text">
    <string>Удалить </string>
   </property>
  </action>
  <action name="actionEdit">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Редактировать</string>
   </property>
  </action>
  <action name="addDeep">
   <property name="text">
    <string>Добавить вглубь</string>
   </property>
  </action>
  <action name="actionSaveK">
   <property name="text">
    <string>Сохранить</string>
   </property>
   <property name="visible">
    <bool>false</bool>
   </property>
  </action>
  <action name="actionSaveKas">
   <property name="text">
    <string>Сохранить как...</string>
   </property>
  </action>
  <action name="actionNewK">
   <property name="text">
    <string>Новый курс</string>
   </property>
  </action>
  <action name="actionup">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="marks.qrc">
     <normaloff>:/arrow_up.svg</normaloff>
     <disabledoff>:/arrow_upDIS.svg</disabledoff>:/arrow_up.svg</iconset>
   </property>
   <property name="text">
    <string>Переместить вверх</string>
   </property>
  </action>
  <action name="actionDown">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="marks.qrc">
     <normaloff>:/arrow_down.svg</normaloff>
     <disabledoff>:/arrow_downDIS.svg</disabledoff>:/arrow_down.svg</iconset>
   </property>
   <property name="text">
    <string>Переместить вниз</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QWebEngineView</class>
   <extends>QWidget</extends>
   <header>qwebengineview.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="marks.qrc"/>
 </resources>
 <connections/>
</ui>
